test/navigation/foreign/ForeignPropertyNavigatorSpec.scala

```diff
@@ -113,6 +114,20 @@ class ForeignPropertyNavigatorSpec extends SpecBase {
         ) mustBe SelectIncomeCountryController.onPageLoad(taxYear, 0, NormalMode)
       }
 
+      "must go from TotalIncomePage to SelectIncomeCountryPage if income is more than £1,000 " +
```

Comment:
Should probably be "must go from TotalIncomePage to **CountriesRentedPropertyPage**..."